scripts/command_line_scghg.py

```diff
@@ -330,8 +336,7 @@ def epa_scghgs(sectors,
 
     # Read generated config    
     master = Path(os.getcwd()) / conf_name
-    with open(master, "r") as stream:
-        conf = yaml.safe_load(stream)
+    conf = read_replace_conf(master)
```

Comment:
looks like `conf` is opened in this func and in the inner func (`epa_scghg`) which seems unnecessary